DigitalLearningSolutions.Data/DataServices/CourseDataService.cs

```diff
@@ -1821,6 +1822,22 @@ THEN CAST(1 AS BIT)
             );
         }
 
+        public bool IsCourseCurrent(int candidateId, int customisationId)

```

Comment:
We already have a method here named 'IsCourseCompleted' that does pretty much the same thing. I feel that we have just negated the existing logic to mean exactly the same thing especially when I see this method consumed in the if else statement in SetTempData() in the controller.

Reply:
Kindly check again, I have fixed the issue another way